.gitlab/functional_test/include.yml

```diff
@@ -5,7 +5,10 @@
 include:
   - .gitlab/functional_test/serverless.yml
   - .gitlab/functional_test/static_quality_gate.yml
-  - .gitlab/functional_test/regression_detector.yml
+  - .gitlab/functional_test/regression_detector.yml  
+  - .gitlab/functional_test/regression_detector_otel_vs_oss.yml
+  - .gitlab/functional_test/regression_detector_oss_vs_otel.yml
```

Comment:
Are you planning on adding this file later, or is it a duplicate?

Reply:
This was a test. As discussed, this PR will be eventually closed.